docs/aws/services/dynamodb-pricing.md

```diff
@@ -0,0 +1,121 @@
+| Read Operation | `GetItem, Scan, Query`                                      | Scans return the entire contents of a table; Queries allow a much faster & cheaper read of a subsection of the table                                        | [Docs](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/bp-query-scan.html)                                        |
+| Indexes        | `None, Local Secondary Index, Global Secondary Index`       | Indexes allow an alternate set of `partition_key` + `sort_key` to be used for queries                                                                       | [Overview](https://www.dynamodbguide.com/secondary-indexes/)                                                                       |
+
+
+## Billing Mode
+
+"Use on-demand until it hurts" - [Alex DeBrie, quoting Jared Short](https://twitter.com/geoff_baskwill/status/1421181922097737729)
+
+Provisioned Throughput is cheaper if you have a meaningful number of reads/writes *distributed evenly across time*. Any reads/writes above the provisioned threshold *will fail*, so it is not well suited to bursty or unpredictable workloads.
+
+Provisioned Throughput includes optional Auto-Scaling if throughput thresholds are being exceeded ([Docs](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/AutoScaling.html)).
+
+[AWS Free Tier](https://aws.amazon.com/free) includes 25 reads/writes-per-second of Provisioned Throughput (across any # of tables), but *does not* include any On Demand mode usage.
+
+
+## Write Type
+
+Standard writes are relatively straightforward and include single item writes (`table.put_item`) and batch writes (`batch.put_item`).
+
+Transactions (`client.execute_transaction`) group up to 25 writes (or reads, updates, or deletes) together and *guarantee that they succeed or fail together.*
+
+For a given write of an item up to *1 KB* in size:
+
+| Type                 | Cost                       |
+|----------------------|----------------------------|
+| Standard single item | 1 WRU                      |
+| Standard batch       | 1 WRU (per item)           |
+| Transactional        | 2 WRU (2x)                 |
+| Oversize 4 KB item   | 4 WRU (4x, size dependent) |
+
+
+## Read Type
+
+Part of what makes DynamoDB a compelling offering is its hybrid approach to the CAP theorem - it can adjust between eventually and strongly Consistent as needed.
+
+Wherever acceptable to the business needs and current data modeling, it is faster and cheaper to use eventually consistent reads.
+
+That said, some business logic unequivocally dictates strongly consistent reads (for example: an ATM reading a customer's balance).
+
+For a given read of an item up to *4 KB* in size:
+
+| Type                  | Cost       |
+|-----------------------|------------|
+| Eventually Consistent | 0.5 RRU    |
+| Strongly Consistent   | 1 RRU (2x) |
+| Transactional         | 2 RRU (4x) |
```

Comment:
For the first reference to RRU do you mind just spelling it out as `Read Request Unit (RRU)` then use the acronym thereafter to just help educate folks as to what it means?